iroh-net/src/discovery/local_swarm_discovery.rs

```diff
@@ -2,7 +2,36 @@
 //!
 //! This allows you to use an mdns-like swarm discovery service to find address information about nodes that are on your local network, no relay or outside internet needed.
 //! See the [`swarm-discovery`](https://crates.io/crates/swarm-discovery) crate for more details.
-
+//!
+//! If you have [`LocalSwarmDiscovery`] enabled, you can get a list of the locally discovered nodes by filtering a list of `RemoteInfo`s.
```

Comment:
Since this comment might be relevant for future code as well, I remember @flub having a preference for avoiding the `we` and `you` phrasing in code. An alternative could simply be

```suggestion
//! When [`LocalSwarmDiscovery`] is enabled, it's possible to get a list of the locally discovered nodes by filtering a list of `RemoteInfo`s.
```

Reply:
Backing this up with an actual style-guide... style guide says "you" is fine :smile:, but "we", "our" or "us" is not:  https://developers.google.com/style/person

But it's true that I also try and avoid "you".  Maybe that is over the top though.